src/Administration/Resources/app/administration/src/module/sw-first-run-wizard/view/sw-first-run-wizard-introduction/sw-first-run-wizard-introduction.scss

```
@@ -0,0 +1,28 @@
+@import "~scss/variables";
+
+.sw-first-run-wizard-introduction {
+    width: 100%;
+    align-self: flex-start;
+    margin-top: 100px;
+
+    .center {
+        text-align: center;
+    }
+
+    .headline-welcome {
```

```suggestion
    &__welcome-headline {
```

Please use Selectors according to [BEM](https://getbem.com/)